uqcsbot/__init__.py

```diff
@@ -19,9 +20,11 @@ def main():
 
     # Run bot
     if '--dev' in sys.argv:
+        logging.basicConfig(level=logging.DEBUG)
```

Comment:
I would rather the logging level toggle be a separate command, I do all the debugging directly against the slack API and would like the debug level in that circumstance

Reply:
You OK with this being provisional and moving it out once we do nicer arg parsing? (perhaps with `click`). If so, I'll add a `# TODO`, if not I'll change it now.

Reply:
Am fine with fixing it later.